CHANGES.rst

```diff
@@ -45,6 +45,8 @@ Breaking changes in 0.3.0
   setup the cleanup job.
 - ``@attr.s(cmp=False)`` is deprecated and all classes have been moved to
   ``@attr.s(eq=False)``, this release requires attrs version 19.2.0
+- Coordinator now saves the labgrid specific files like ``places.yaml`` and
+  ``resources.yaml`` in its working-dir, Previously it would use ``..``.
```

Comment:
This is not correct. The coordinator always saves these files in its working dir. The change is that the crossbar sets the coordinator working dir to the config directory, instead of one level up.

Reply:
Point taken. Thanks.